topdown/crypto.go

```diff
@@ -160,12 +164,68 @@ func builtinCryptoSha256(a ast.Value) (ast.Value, error) {
 
 func init() {
 	RegisterFunctionalBuiltin1(ast.CryptoX509ParseCertificates.Name, builtinCryptoX509ParseCertificates)
+	RegisterBuiltinFunc(ast.CryptoX509ParseAndVerifyCertificates.Name, builtinCryptoX509ParseAndVerifyCertificates)
 	RegisterFunctionalBuiltin1(ast.CryptoMd5.Name, builtinCryptoMd5)
 	RegisterFunctionalBuiltin1(ast.CryptoSha1.Name, builtinCryptoSha1)
 	RegisterFunctionalBuiltin1(ast.CryptoSha256.Name, builtinCryptoSha256)
 	RegisterFunctionalBuiltin1(ast.CryptoX509ParseCertificateRequest.Name, builtinCryptoX509ParseCertificateRequest)
 }
 
+func getX509CertsFromBytes(bytes []byte) ([]*x509.Certificate, error) {
+	// if the input is not a PEM string, attempt to decode b64
+	if str := string(bytes); !strings.HasPrefix(str, "-----BEGIN CERTIFICATE-----") {
+		var err error
+		bytes, err = base64.StdEncoding.DecodeString(str)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
[nit] I don't think it matters for performance at all, but we're only feeding this `[]byte(someString)` -- we might as well go with
```suggestion
func getX509CertsFromBytes(certs string) ([]*x509.Certificate, error) {
	var decodedCerts []byte
	// if the input is not a PEM string, attempt to decode b64
	if strings.HasPrefix(certs, "-----BEGIN CERTIFICATE-----") {
		decodedCerts = []byte(certs)
	} else {
		var err error
		decodedCerts, err = base64.StdEncoding.DecodeString(str)
		if err != nil {
			return nil, err
		}
	}
```
Well, not a big difference. Just do as you see fit 😉 

Let's try to avoid calling variables by their type (`var bytes []byte`) but use something indicating their contents. 😃

Reply:
Makes sense, I'll update this as well as the function name.